Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -401,6 +461,34 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this.color = resolvedColor;
         this.dispatchEventToListeners(WI.ColorPicker.Event.ColorChanged, {color: this._color, variableName});
     }
+
+    _updateColorContrast(color)
+    {
+        if (this._activeSwatch && color) {
+            this._activeSwatch.value = color;
+            this._activeSwatch.element.style.backgroundColor = color;
+            if (this._contrastObject.active === "background-color")
+                this._contrastObject.propertyBackgroundColor = color;
+            else
+                this._contrastObject.propertyForegroundColor = color;
+        }
+
+        if (this._contrastScore) {
+            let contrast = WI.ColorPicker.accessabilityContrast(this._contrastObject.propertyBackgroundColor, this._contrastObject.propertyForegroundColor);
```

Comment:
Typo: `accessabilityContrast` -> `accessibilityContrast`

A more descriptive name would be `getColorContrastAccessibilityRatio()`, or something similar.
https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio